docs/admin/README.md

```diff
@@ -8,6 +8,7 @@ As a infrastructure administrator, I want to:
  * [extend this example application.](extending-example.md)
  * [verify changes to this application.](testing-changes.md)
  * [perform management commands.](management-commands.md)
+ * [see telemetry for my application.](opentelemetry.md)
```

Comment:
issue: opentelemetry.md missing? I don't this in a separate PR or HEAD.

Reply:
I didn't commit the doc and now it's lost. Will recreate

Reply:
doc recreated